one_to_hundred.rb

```diff
@@ -0,0 +1,3 @@
+for i in 1..100
+  puts"#{i}"
```

Comment:
No need to use **" "** and **{ }** if you wish to print only number. This is used when number in used in between the string.